### DISCUSSION TITLE
Not working tmute tban

### DISCUSSION THREAD

**Discussion by instant992** [AmanoTeam/EduuRobot#325] [Q&A] [answered]
The commands for temporary mute and temporary ban do not work. 
How to fix it?

**Comment by alissonlauffer** [comment, marked as answer]
Should be fixed now. tban apparently is banning permanently (Looks like a Pyrogram bug, I guess?). tmute is now working fine.

**Reply by instant992** [reply]
Also, clearing service messages does not work, it gives an error in the console

**Reply by alissonlauffer** [reply]
Fixed too. Also fixed some other missed cases. Thanks for reporting!

**Comment by instant992** [comment]
Is it possible to embed your custom commands in the bot, and if so, how?
It's just that the bot is good, but there are relatively few commands

**Reply by alissonlauffer** [reply]
Well, you can easily add commands to your own clone by adding files to the `eduu/plugins/` directory. Contributions are also welcome.

If you mean with `@EduuRobot`, you can create custom filters/notes too.
